### PR TITLE
Added a fix for noobaa-standalone placement configuration

### PR DESCRIPTION
**Pull request by achouhan09**
Currently, default placement is used if noobaa-standalone placement is not found. We want to use noobaa-core if noobaa-standalone placement is not provided or default otherwise.

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2260550

### CODE REVIEW

File: controllers/storagecluster/placement.go [hunk 1]
```diff
@@ -18,7 +18,13 @@ func getPlacement(sc *ocsv1.StorageCluster, component string) rookCephv1.Placeme
 	if ok {
 		(&in).DeepCopyInto(&placement)
 	} else {
-		in := defaults.DaemonPlacements[component]
+		// fallback to 'noobaa-core' if 'noobaa-standalone' placement not found, or use default placement.
+		if component == "noobaa-standalone" {
```
Comment (iamniting) [inline controllers/storagecluster/placement.go:22]: I don't think we should add this logic here, This is a generic func, We should check in the calling site and call accordingly.

**Review by iamniting — CHANGES_REQUESTED** [review]

File: controllers/storagecluster/placement.go (hunk 1 above)
Comment (achouhan09) [reply controllers/storagecluster/placement.go]: Thanks, sure will update accordingly.

**Review by iamniting — APPROVED** [review]

**Pull request merged**